app/scripts/proactive/config.js

```diff
@@ -95,17 +95,17 @@ define(function () {
         },
         languages_content_type: {
             "scalaw" : "text/x-scala",
-            "groovy" : "application/x-groovy",
-            "javascript" : "application/javascript",
-            "python" : "application/x-python",
-            "cpython" : "application/x-python",
-            "ruby" : "application/x-ruby",
+            "groovy" : "text/x-groovy",
+            "javascript" : "text/javascript",
+            "python" : "text/x-python",
+            "cpython" : "text/x-python",
+            "ruby" : "text/x-ruby",
             "r" : "text/x-R",
             "bash" : "application/x-sh",
```

Comment:
depending on the logic upon which you changed the other from `application` to `text` MIME types, these ones should also follow that logic

Reply:
After looking at which content-types code mirror understands (https://codemirror.net/mode/), some are 'text/' (groovy for example) and some are 'application' (powershell for example)

Reply:
👍